packages/react-scripts/package.json

```diff
@@ -20,6 +20,7 @@
   "files": [
     "bin",
     "config",
+    "PATENTS",
```

Comment:
Thanks! No worries, some of the legal related things unfortunately need some special treatment :)